massa-models/src/output_event.rs

```diff
@@ -19,7 +19,7 @@ impl Display for SCOutputEvent {
 }
 
 /// Context of the event (not generated by the user)
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
```

Comment:
in which case do we need PartialEq for such a complex structure ?

Reply:
It is used in unit test so PartialEq is now only derived when feature "test-exports" is enabled